src/components/icon-as-button/_icon-as-button.scss

```diff
@@ -27,16 +27,16 @@ $includeHtml: false !default;
 
 @mixin colorTheme($modifierName, $color, $colorActive, $actionForegroundColor: $iconAsButtonActionColor) {
   &.sg-icon-as-button--#{$modifierName} {
-    fill: $color;
+    color: $color;
```

Comment:
@katarzynatobis @clxandstuff probably we need to add story for this deprecated button.Not necessary in this PR.

Previously we skipped this component in storybook because we didn't  know it would be useful for chromatic.